src/js/vic-v2/components/VeteranIDCard.jsx

```diff
@@ -1,7 +1,106 @@
 import React from 'react';
 
-export default function VeteranIDCard() {
-  return (
-    <div style={{ paddingTop: '95px', backgroundColor: 'gray', height: '200px', width: '300px' }}></div>
-  );
+const FONT_FAMILY = 'Arial, Helvetica, sans-serif';
+// sets the aspect ratio of svg
+const VIEW_BOX = '0 0 2048 1284';
+// height of font top to bottom bounding box in the above viewbox
+const LARGE_FONT_SIZE = 90;
+const SMALL_FONT_SIZE = 60;
+
+const imagePaths = {
+  'Coast Guard': '/img/vic-cg-emblem.png',
+  'Air Force': '/img/vic-air-force-coat-of-arms.png',
+  Army: '/img/vic-army-symbol.png',
+  Navy: '/img/vic-navy-emblem.png',
+  'Marine Corps': '/img/vic-usmc-emblem.png',
+  previewBackground: '/img/vic-preview-background.svg',
+  VASeal: '/img/vic-va-seal.png'
+};
+
+function getIsLongName(name) {
+  try {
+    return document
+      .createElement('canvas')
+      .getContext('2d')
+      .measureText(name)
+    // manually determined max width
+      .width > 120;
+  } catch (e) {
+    return true;
+  }
 }
+
+const VeteranIDCard = ({
+  veteranFirstName,
+  veteranLastName,
+  veteranBranch,
+  veteranID
+}) => (
+  // svg preserves aspect ratio
+  <svg
+    fill={'white'}
```

Comment:
Curious why you do this vs `fill="white"` here and elsewhere for string props.

Reply:
Habit. Will update.